cmd/client-s3.go

```diff
@@ -894,12 +884,14 @@ func (c *s3Client) Stat(isIncomplete bool) (*clientContent, *probe.Error) {
 			objectMetadata.Time = objectStat.LastModified
 			objectMetadata.Size = objectStat.Size
 			objectMetadata.Type = os.FileMode(0664)
+			objectMetadata.Metadata = map[string][]string{}
```

Comment:
why should we try to find the given object by listing the objects in the bucket for fetching stat information? We can directly call `c.api.StatObject(bucket, object)`, right?